test/integration/transport.js

```diff
@@ -95,6 +117,42 @@ describe('given configurations for 10 nodes with address "127.0.0.1", WS ports 5
 					});
 				});
 
+				it('there should exactly 10 listening connections for 500[0-9] ports', done => {
```

Comment:
10 or 9, it doesn't correlate with the expectation of 180 in the test below.

Reply:
There should be 10 listening connections as we started 10 instances.